app/code/Magento/LayeredNavigation/view/frontend/templates/layer/filter.phtml

```diff
@@ -12,38 +11,20 @@
 /** @var \Magento\LayeredNavigation\ViewModel\Layer\Filter $viewModel */
 $viewModel = $block->getData('product_layer_view_model');
 ?>
-
 <ol class="items">
     <?php foreach ($filterItems as $filterItem): ?>
         <li class="item">
-            <?php if ($filterItem->getCount() > 0): ?>
-                <a
-                    href="<?= $escaper->escapeUrl($filterItem->getUrl()) ?>"
-                    rel="nofollow"
-                ><?= /* @noEscape */ $filterItem->getLabel() ?><?php
-                if ($viewModel->shouldDisplayProductCountOnLayer()): ?><span
-                        class="count">
-                            <?= /* @noEscape */ $localeFormatter->formatNumber((int) $filterItem->getCount()) ?><span
-                            class="filter-count-label"><?php
-                            if ($filterItem->getCount() == 1): ?>
-                                <?= $escaper->escapeHtml(__('item')) ?><?php
-                            else:
-                                ?><?= $escaper->escapeHtml(__('item')) ?><?php
-                            endif;?></span></span>
-                    <?php endif; ?></a>
-            <?php else: ?>
-                <?= /* @noEscape */ $filterItem->getLabel() ?><?php
-                if ($viewModel->shouldDisplayProductCountOnLayer()): ?><span
-                        class="count">
-                            <?= /* @noEscape */ $localeFormatter->formatNumber((int) $filterItem->getCount()) ?><span
-                            class="filter-count-label"><?php
-                            if ($filterItem->getCount() == 1): ?>
-                                <?= $escaper->escapeHtml(__('items')) ?><?php
-                            else:
-                                ?><?= $escaper->escapeHtml(__('items')) ?><?php
-                            endif;?></span></span>
+            <a href="<?= $escaper->escapeUrl($filterItem->getUrl()) ?>" rel="nofollow">
+                <?= /* @noEscape */ $filterItem->getLabel() ?>
+                <?php if ($viewModel->shouldDisplayProductCountOnLayer()): ?>
+                    <span class="count">
+                        <?= /* @noEscape */ $localeFormatter->formatNumber((int) $filterItem->getCount()) ?>
+                        <span class="filter-count-label">
+                            <?= $filterItem->getCount() <= 1 ? $escaper->escapeHtml(__('item')) : $escaper->escapeHtml(__('items')) ?>
```

Comment:
Great optimisation, makes it a lot easier to read!

However, shouldn't this line be:

```php
<?= $filterItem->getCount() !== 1 ? $escaper->escapeHtml(__('items')) : $escaper->escapeHtml(__('item')) ?>
```

Otherwise we would get:
- 0 item
- 1 item
- 2 (or more) items

The correct way (at least in English, but there are some languages where this also wouldn't be correct):
- 0 item**s**
- 1 item
- 2 (or more) items

Reply:
Good spot - added.